workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/SpatialOSEntity.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Improbable.Worker;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    /// <summary>
+    ///     Wrapper around an ECS Entity instance representing a SpatialOS Entity used for interacting with the the
```

Comment:
Rephrase for clarity: 
"Retrieve information about a SpatialOS Entity instance from a Unity ECS Entity instance."
cc @ElleEss for terminology.